tools/flight_recorder/components/builder.py

```diff
@@ -284,11 +300,17 @@ def build_collectives(
                     check_result, input_numel, output_numel = check_size_alltoall(
                         alltoall_cases
                     )
+                    # We don't log the input/output sizes for alltoall so we don't consider the size mismatch as an error for now.
+                    check_result = True
                     if not check_result:
```

Comment:
This `if` feels backwards?
I would have expected you to do something like:
```
check_result = False
if check_result:
```

Reply:
maybe I can rename it to `fail_check`